src/ensembl/src/shared/image-button/ImageButton.test.tsx

```diff
@@ -13,62 +14,88 @@ describe('<ImageButton />', () => {
     it('has a buttonStatus set by default', () => {
       const wrapper = mount(<ImageButton />);
 
-      expect(wrapper.props().buttonStatus).toEqual(ImageButtonStatus.DEFAULT);
-    });
-
-    it('respects the buttonStatus prop', () => {
-      const wrapper = mount(
-        <ImageButton buttonStatus={ImageButtonStatus.ACTIVE} />
-      );
-
-      expect(wrapper.props().buttonStatus).toEqual(ImageButtonStatus.ACTIVE);
+      expect(wrapper.prop('buttonStatus')).toEqual(ImageButtonStatus.DEFAULT);
     });
   });
 
   describe('prop description', () => {
     it('has a description set by default', () => {
       const wrapper = mount(<ImageButton />);
 
-      expect(wrapper.props().description).toEqual('');
+      expect(wrapper.prop('description')).toEqual('');
     });
 
     it('respects the description prop', () => {
       const wrapper = mount(<ImageButton description={'foo'} />);
 
-      expect(wrapper.props().description).toEqual('foo');
+      expect(wrapper.prop('description')).toEqual('foo');
     });
   });
 
   describe('prop image', () => {
-    it('has a image set by default', () => {
+    it('has an image set by default', () => {
       const wrapper = mount(<ImageButton />);
 
-      expect(wrapper.props().image).toEqual('');
+      expect(wrapper.prop('image')).toEqual('');
     });
 
-    it('respects the image prop', () => {
-      const wrapper = mount(<ImageButton image={'foo.svg'} />);
+    it('renders and img tag if a path to an image file is passed', () => {
+      const wrapper = mount(<ImageButton image={'foo.png'} />);
 
-      expect(wrapper.props().image).toEqual('foo.svg');
+      expect(wrapper.find(ImageHolder).find('img')).toHaveLength(1);
```

Comment:
And make sure that <img> has the expected source (string passed in the image prop).

Reply:
That's done